Source/carthage/Update.swift

```diff
@@ -90,7 +95,9 @@ public struct UpdateCommand: CommandProtocol {
 		return options.loadProject()
 			.flatMap(.merge) { project -> SignalProducer<(), CarthageError> in
 
-				let checkDependencies: SignalProducer<(), CarthageError>
+                project.useNetrc = options.useNetrc
```

Comment:
@tmspzz  I have been trying to modify `Project.func downloadBinaryFrameworkDefinition(binary: BinaryURL) -> SignalProducer<BinaryProject, CarthageError>` method by adding additional `useNetrc: Bool` param. However there are quite a few places where this method is used into and I was not able to get the `netrc` value from `update` command options in that cases.

Reply:
Well, the flag should be passed down to the appropriate places all the way or defaulter to false if not relevant. Can you point me to where you're unable to access the flag?

Reply:
I just started by modifying a mentioned function to:

```
// Project.swift file

func downloadBinaryFrameworkDefinition(binary: BinaryURL, useNetrc: Bool) -> SignalProducer<BinaryProject, CarthageError>
```

and eventually the adding `netrc: Bool` parameter to 👇 became an issue:

```
// Project.swift file

func versions(for dependency: Dependency) -> SignalProducer<PinnedVersion, CarthageError>
```

That's why I decided to add a property to the `Project`.

Reply:
Hey @tmspzz 👋 

Just wanted to kindly ask if you maybe got a chance to have a look at this one? 🙏 Please feel free to ask more questions if something's not clear 😉 Thanks! 🙇